app/views/training/pages/certificate.html.slim

```diff
@@ -16,44 +16,49 @@
         =< govuk_link_to 'My account', user_path
         | .
 
-h3.govuk-heading-s Certificate of achievement
+.govuk-grid-row
+  .govuk-grid-column-three-quarters-from-desktop id=('award' if request.original_url.end_with?('.pdf'))
 
-- if module_progress.completed?
-  h1.govuk-heading-xl.data-hj-suppress= current_user.name
-  p has completed the following module:
-- else
-  h1.govuk-heading-xl Your name will appear here
+    h3.govuk-heading-s Certificate of achievement
 
-h2.govuk-heading-m= mod.title
```

Comment:
add classes: cert-32 cert-bold
Needs a custom margin at the bottom of 0px so it is in effect underlined by the new HR below it